metadata-ingestion/src/datahub/ingestion/extractor/json_schema_util.py

```diff
@@ -620,6 +658,29 @@ def get_fields_from_schema(
                 base_field_path=FieldPath(is_key_schema=is_key_schema),
             )
 
+    @classmethod
+    def extract_descriptions_from_schema(cls, schema: Dict[str, Union[str, Dict]]) -> List[Optional[str]]:
+        """
+        Extracts descriptions from a JSON schema.
+
+        Args:
+            schema (Dict[str, Union[str, Dict]]): The JSON schema.
```

Comment:
is this the same method as above?

Reply:
Hey @hsheth2, Yeah. Oppsy. Let me fix things up here in this PR. Give me sometime please.